lib/mongodb.js

```diff
@@ -39,6 +40,7 @@ function ObjectID(id) {
   }
 }
 
+exports.generateMongoDBURL = generateMongoDBURL;
```

Comment:
Why do we need to use export on the helper functions? Correct me if I am wrong, but I think they are only used in this file. Same goes for `ObjectID`